Enigmatry.Entry.MediatR.Tests/ValidationBehaviorPipelineFixture.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Enigmatry.Entry.MediatR.Tests.Samples;
+using FluentValidation;
+using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Enigmatry.Entry.MediatR.Tests;
+
+[Category("unit")]
+public class ValidationBehaviorPipelineFixture
```

Comment:
We didn't have any tests for the ValidationBehavior, so this was a good moment to add them. First to verify that we indeed have a bug and then to fix the bug.